File: app/src/main/java/com/example/nakama/testbottomsheet/model/TextViewHolderModel.java
package com.example.nakama.testbottomsheet.model;

import com.example.nakama.testbottomsheet.viewholders.TextViewHolder;

/**
 * Created by Hendri on 24/05/18.
 */
public class TextViewHolderModel extends BaseModel {

    String text;

    public TextViewHolderModel(String text) {
        this.text = text;
    }

    @Override
    public int getType() {
        return TextViewHolder.LAYOUT;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }
}

File: app/src/main/java/com/example/nakama/testbottomsheet/model/TextViewHolderModel.kt
package com.example.nakama.testbottomsheet.model

import com.example.nakama.testbottomsheet.viewholders.TextViewHolder

/**
 * Created by Hendri on 24/05/18.
 */
class TextViewHolderModel(var text: String) : BaseModel() {

    override val type: Int
        get() = TextViewHolder.LAYOUT
}
